booking-flight-service/src/test/resources/schema.sql

```diff
@@ -0,0 +1,110 @@
+-- MySQL Script generated by MySQL Workbench
```

Comment:
Remove MySQL workbench generated comments, and remove unnecessary tables that aren't being used from the schema